### PR TITLE
kubectl run creates a Deployment by default, fix the docs.

### PR DESCRIPTION
**Pull request by madhusudancs**
Ref: #23231

cc @bgrant0607 @martinp

### CODE REVIEW

File: examples/simple-nginx.md [hunk 1]
```diff
@@ -57,13 +57,13 @@ kubectl get pods
 You can also see the replication controller that was created:
```
Comment (janetkuo) [inline examples/simple-nginx.md]: s/replication controller/deployment/g everywhere in this doc

Comment (madhusudancs) [inline examples/simple-nginx.md]: Oops, I saw them and forgot to fix them. Sorry about that. Fixed them now.

**Comment by madhusudancs** [comment]
@janetkuo PTAL.

**Comment by janetkuo** [comment]
#23348 is fixing the same doc too.

**Comment by madhusudancs** [comment]
@janetkuo @pwittrock Sorry, didn't see #23348. It was sent before this PR and is arguably better, so I am closing this PR.

@janetkuo Thanks for reviewing though.

**Pull request closed without merging**